src/index.ts

```diff
@@ -188,8 +188,11 @@ export class ActionsAPI {
     overwrite: boolean = false
   ): Promise<any> {
     // Example: PUT /ws/:workspaceId/:name
+    // Strip path, keep only the file name
+    const filenameOnly = name.split(/[/\\]/).pop()!;
```

Comment:
@joshhaug do we ever expect backslashes `\` in the path or can this be limited to forward slashes `/` only? just wondering if this could accidentally hit eg. escaped strings with backslashes in them.